Adding T cruzi section to new reports

srouseTxbiomed opened the pull request:
#### Rationale
Adding report for T Cruzi new positive to new surveillance section of More reports

#### Related Pull Requests
* NA

#### Changes
* Added new report section surveillance in  SNPRC_EHRModule.java

srouseTxbiomed committed:
Report summary data for T cruzi working
03.31.2021
srr

srouseTxbiomed committed:
Working queries for TC report
04.16.2021
srr

srouseTxbiomed committed:
Working queries for TC report
Update 14:54
04.16.2021
srr

srouseTxbiomed committed:
Adding report for T Cruzi new positive
 Added new surveillance section of More reports
srr

srouseTxbiomed committed:
Merge branch 'develop' into fb_snprc_TcruziSummary

spamhurts reviewed (commented):
I don't see the indeterminate results on the ReportTcruziSummary report. Am I missing something?

srouseTxbiomed commented:
I forgot I had already added the summary report in previous commit.  I moved them to a different changelist, but I can add them back.  The only report referenced under More Reports is the New Positives  (last 12 months) report Dr. Elmore requested.
